barretenberg/cpp/src/barretenberg/ultra_honk/merge_prover.cpp

```diff
@@ -41,10 +40,6 @@ template <typename Flavor> MergeProver_<Flavor>::MergeProof MergeProver_<Flavor>
     std::array<Polynomial, NUM_WIRES> T_prev = op_queue->construct_previous_ultra_ops_table_columns();
     std::array<Polynomial, NUM_WIRES> t_current = op_queue->construct_current_ultra_ops_subtable_columns();
 
-    // TODO(#723): Cannot currently support an empty T_prev. Need to be able to properly handle zero commitment.
```

Comment:
723 is a closed issue, we can handle zero commitments in some of the scenarios, the issues remain around this https://github.com/AztecProtocol/barretenberg/issues/993
at least this my understanding for a preliminary investigation